230 Hidden lookup and datepicker icon

tmcastro opened the pull request:
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
- Added validation to hide icons if inputs are hidden
- http://localhost:4000/components/lookup/test-hidden-lookup-input.html
- http://localhost:4000/components/datepicker/test-hidden-datepicker-input.html

**Related github/jira issue (required)**:
Closes #230 

**Steps necessary to review your pull request (required)**:
- Upon loading of the test page, both the input and icons of the control should now be hidden

clepore commented:
@tmcastro Can you please update the description with the correct testing instructions for this pull request? QA tried to test it, but the 2 test files listed do not exist and they weren't sure which test pages this sentence referred to:
> Upon loading of the test page, both the input and icons of the control should now be hidden

(cc @tmcconechy, @CindyMercadoReyes )

tmcconechy commented:
As per my instructions there is now no way to test it visually. Its covered only by a functional test.
So save to move to done.

tmcastro commented:
Hi guys, sorry about that. I've removed the 2 test pages and just added 2 new test scripts for both lookup and datepicker (Tim's instructions). The scripts should be working fine. Thanks!
